src/pages/home/hero-area/hero-area.jsx

```diff
@@ -1,25 +1,52 @@
-import React, { useState } from "react";
-import PropTypes from "prop-types";
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import _ from 'lodash';
+import moment from 'moment';
 
-import Button from "react-bootstrap/Button";
-import FormControl from "react-bootstrap/FormControl";
+import Button from 'react-bootstrap/Button';
+import FormControl from 'react-bootstrap/FormControl';
 
-import backgroundImage from "img/movies.png";
+import AddMovieModal from 'components/add-movie-modal';
 
-import styles from "./hero-area.module.scss";
+import * as movieActions from 'resources/movies/movies.actions';
+import { DEFAULT_DATE_FORMAT } from 'constants/date-formats';
+
+import backgroundImage from 'img/movies.png';
+
+import styles from './hero-area.module.scss';
 
 function HeroArea(props) {
-  const [searchString, setSearchString] = useState("");
+  const dispatch = useDispatch();
+
+  const [searchString, setSearchString] = useState('');
+  const [isAddMovieModalOpened, setIsAddMovieModalOpened] = useState(false);
+
+  const onOpenAddMovieModal = () => {
+    setIsAddMovieModalOpened(true);
+  }
+
+  const onCloseAddMovieModal = () => {
+    setIsAddMovieModalOpened(false);
+  }
+
+  const onSubmit = (data) => {
+    const movie = {
+      ...data,
+      id: _.random(100000, 999999),
```

Comment:
It is possible to use [Math.random](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) to avoid using lodash.